MAVProxy/modules/mavproxy_param.py

```diff
@@ -601,7 +605,7 @@ def add_new_target_system(self, sysid):
         if self.continue_mode and self.logdir is not None:
             parmfile = os.path.join(self.logdir, fname)
             if os.path.exists(parmfile):
-                mpstate.mav_param.load(parmfile)
+                self.mpstate.mav_param.load(parmfile)
```

Comment:
I'm wondering if we should really be passing `check=False` through unconditionally here.  I think the expectation is that nothing has changed, but I'm really not sure why we'd go and change the vehicle state when we're doing a `--continue`

Reply:
I don't know what the real intent of continue is.  Maybe not so many users considering it may have been broken anyway.  Will revert to original.
Many bugs in this PR.  Might close and re-open a fresh one.  Thanks for the review.